scene/gui/label.h

```diff
@@ -63,7 +63,7 @@ class Label : public Control {
 		OVERRUN_TRIM_CHAR,
 		OVERRUN_TRIM_WORD,
 		OVERRUN_TRIM_ELLIPSIS,
-		OVERRUN_TRIM_WORD_ELLIPSIS,
+		OVERRUN_TRIM_WORD_ELLIPSIS
```

Comment:
We actually use trailing commas on purpose to avoid diffs such as you're getting in `servers/text_server.h`, so this shouldn't be removed.

Reply:
Done.